Makefile

```diff
@@ -140,7 +141,12 @@ container-tests: pkg/apis/wksprovider/machine/scripts/scripts_vfsdata.go pkg/api
 	go test -count=1 ./test/container/...
 
 integration-tests-container: cmd/wksctl/wksctl cmd/controller/.uptodate
-	IMAGE_TAG=$(IMAGE_TAG) go test -v -timeout 20m ./test/integration/container/...
+	IMAGE_TAG=$(IMAGE_TAG)
+	APK_INDEX='https://https:dl-cdn.alpinelinux.org/alpine/v3.11/main/x86_64/APKINDEX.tar.gz https://dl-cdn.alpinelinux.org/alpine/v3.11/community/x86_64/APKINDEX.tar.gz'
+
+	for node_os in $${OSS:-"centos ubuntu"}; do \
+		NODE_OS="$${node_os} go test -v -timeout 40m ./test/integration/container/..."; \
```

Comment:
This looks wrong to me. Shouldn't it be: `NODE_OS="$${node_os}" go test -v -timeout 40m ./test/integration/container/...; \`?

Reply:
This has been updated.